scripts/main.js

```diff
@@ -1,31 +1,75 @@
-const createLog = (event) => {
+import DeleteButton from "./deleteWorkout.js";
+
+const fields = [
+  {
+    id: "time-input",
+    name: "time",
+  },
+  {
+    id: "workout-input",
+    name: "workout",
+  },
+  {
+    id: "date-input",
+    name: "date",
+  },
+  {
+    id: "action",
+    name: "action",
+  },
+];
+
+let totalTime = 0;
+
+const getWorkoutData = () => {
+  const data = {};
+  fields.forEach((field) => {
+    const element = document.querySelector(`#${field.id}`);
+    if(element !== null){
```

Comment:
Dica:

Nesse caso pode omitir o `!== null`, e deixar somente:

```suggestion
    if(element){
```

O javascript vai rodar uma expressão Boolean() nesse valor e retornar um bool

ex:
![image](https://user-images.githubusercontent.com/15300776/119403467-c7f83600-bcb4-11eb-890f-ee2dd0fc4bae.png)